libs/core/src/components/grid/grid.test.ts

```diff
@@ -18,11 +18,39 @@ describe('GdsGrid', () => {
     expect(slot).to.exist
   })
 
-  it('sets default properties', () => {
-    const element: GdsGrid = document.createElement('gds-grid') as GdsGrid
-    expect(element.columns).to.be.undefined
-    expect(element.gap).to.be.undefined
-    expect(element.padding).to.be.undefined
-    expect(element.autoColumns).to.be.undefined
+  it('sets columns with single value correctly', async () => {
+    ;(element.columns as any) = '2'
```

Comment:
There is something wrong with Prettier. I get these extra `;` that the linting doesn't like... Not sure what causes this.

Reply:
Those are required when a line begins with parenthesis. Otherwise it could be interpreted as an invocation of the previous line by the JS parser.